tools/pyi/gen_pyi.py

```diff
@@ -179,7 +179,7 @@ def arg_to_type_hint(arg):
 
 
 binary_ops = ('add', 'sub', 'mul', 'div', 'pow', 'lshift', 'rshift', 'mod', 'truediv',
-              'matmul', 'floordiv', 'floor_divide'
+              'matmul', 'floordiv', 'floor_divide',
```

Comment:
@mruberry I think this generating a type hint for "__floor_divide__" but floor_divide doesn't seem to be a supported Python magic method, is that intended?

Reply:
I think your right. I'll look at removing it. Thanks @vivekmig!